docs/docs/graphql/permissions.mdx

```diff
@@ -30,107 +30,111 @@ The rule above make it so users can only select posts where the value of `user_i
 
 ## What is `x-hasura-user-id`?
 
-`x-hasura-user-id` is a permission variable that is used to create permission rules in Hasura. The permission variable comes from the [access token](/authentication#access-tokens) that signed-in users have.
+`x-hasura-user-id` is a permission variable that is used to create permission rules in Hasura. The permission variable comes from the [access token](/authentication/tokens#access-token) that authenticated users have.
 
-The `x-hasura-user-id` permission variable is always available for all signed-in users. You can add [custom permission variables](#custom-permission-variables) to create more complex permission rules unique to your project.
+The `x-hasura-user-id` permission variable is always available for all authenticated users. You can add [permission variables](#permission-variables) to create more complex permission rules unique to your project.
 
-## Custom Permission Variables
+## Permission Variables
 
-You can add custom permission variables in the Nhost console under **Users** and then **Roles and permissions**. These permission variables are then available when creating permissions for your GraphQL API in the Hasura console.
+You can add permission variables in the Nhost Dashboard under **Settings -> Roles and permissions**. These permission variables are automatically added to users' [access tokens](/authentication/tokens#access-token). This way, permission variables are available when creating permissions for your GraphQL API in the Hasura Console.
 
 ![Permission Variables](/img/permission-variables-preview.png)
 
-**Example:** Let's say you add a new permission variable `x-hasura-organisation-id` with path `user.profile.organisation.id`. This means that Nhost Auth will get the value for `x-hasura-organisation-id` by internally generating the following GraphQL query:
+**Example:** Let's say you add a new permission variable `x-hasura-company-id` with path `user.company.id`. This means that Nhost Auth will get the value for `x-hasura-company-id` by internally generating the following GraphQL query:
 
 ```graphql
 query {
-  user(id: "<User's ID>") {
+  user(id: "<user-id>") {
+    company {
+      id
+    }
+  }
+}
+```
+
+### Arrays
+
+You must add `[]` at the end of the path for permission variables that are arrays.
+
+**Example**: Let's say you have a permission variable `x-hasura-organization-ids`, the path should be, e.g., `user.profile.organizations.id[]`.
```

Comment:
Is `oranizations` in plural in `user.profile.organizations.id\[\]\` correct?

Reply:
I don't see us using `oranizations` anywhere